src/System.Text.Encodings.Web/src/System/Text/Encodings/Web/EncoderExtensions.cs

```diff
@@ -4,7 +4,7 @@
 using System;
 using System.IO;
 
-namespace System.Text.Encodings.Web
+namespace Microsoft.Framework.WebEncoders
```

Comment:
> so I moved them to a "to be deleted" namespace

Ah, ok, thanks, that part wasn't clear to me.  So anything in this namespace is temporary, will be deleted prior to release, and should be ignored.